runtime/doc/news.txt

```diff
@@ -154,13 +154,27 @@ UI
 • |CompleteDone| now sets the `reason` key in `v:event` which specifies the reason
   for completion being done.
 
+• Added |vim.net.fetch()| for requests.
+
+• Added |vim.net.download()| for file downloads.
+
 ==============================================================================
 CHANGED FEATURES                                                 *news-changed*
 
 These existing features changed their behavior.
 
 • N/A
 
+• `:e, :w, :[range]w!, and :r https://*,http://*,ftp://*,scp://*` can now
+  use |vim.net.fetch()|. Opt-in via >lua
+      vim.g.lua_net_enable = true
+<
+• `:e https://*,http://*,ftp://*,scp://*` now automatically matches
+  filetype using the url. Optionally, you can enable (slower) automatic
+  full-text filetype matching for editing remote files that don't have an
+  extension (for example https://neovim.io/ -> `ft=html`) via >lua
+      vim.g.lua_net_ft_full = true
+<
```

Comment:
We should avoid adding configuration variables for this wherever possible. Since the decision has already been made that NetRW *delendam est* and we are very early in the dev cycle, we should just enable this by default (with a temporary(!) opt-out, hopefully to be removed by the time 0.11 is released).

Also, why would someone _not_ want actual filetype matching? How much slower are we talking about -- milliseconds or minutes? I think the basic expectation (that we should meet) is that remote editing behaves just like local editing. This means we should just do full matching (with possibly an "gimme _nao_, I know what I'm doing" fast option -- again, hopefully to be removed if we can iron out the issues that make it desirable).

Reply:
> How much slower are we talking about -- milliseconds or minutes?

I'm gonna disable this for now. If there are performance problems in the future, I'm gonna add it again